implementations/micrometer-registry-signalfx/src/main/java/io/micrometer/signalfx/SignalFxMeterRegistry.java

```diff
@@ -216,4 +243,30 @@ protected void publish() {
     protected TimeUnit getBaseTimeUnit() {
         return TimeUnit.SECONDS;
     }
+
+    private DistributionStatisticConfig updateConfig(
+            DistributionStatisticConfig distributionStatisticConfig) {
+        // Add the +Inf bucket since the "count" resets every export.
+        double[] sla = distributionStatisticConfig.getServiceLevelObjectiveBoundaries();
+        if (sla == null || sla.length == 0) {
+            return distributionStatisticConfig;
+        }
+        double[] newSLA = sla;
+        if (!isPositiveInf(sla[sla.length - 1])) {
+            newSLA = Arrays.copyOf(sla, sla.length + 1);
+            newSLA[newSLA.length - 1] = Double.MAX_VALUE;
+        }
+        return DistributionStatisticConfig.builder()
+                // Set the expiration duration for the histogram counts to be effectively a lifetime.
+                // Without this, the counts are reset every expiry duration.
```

Comment:
SignalFx can handle this for histogram buckets, right (including server restart and cumulative value reset)?

Reply:
Yes signalfx can handle "Cumulative Counters". Ideally, would prefer the bucket counts to reset every "report" (not every x time), essentially delta counters, but that is not the behavior I can get, so this is good enough.